apps/ngx-env-demo/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --host 0.0.0.0",
```

Comment:
I don't know how to reproduce this issue.
This should work using just `ng serve`, this looks like a workaround.

Reply:
I don't know Will remove it for now maybe it just in my machine